### DISCUSSION TITLE
Inserting integer values into query

### DISCUSSION THREAD

**Discussion by vaishnaray** [Budibase/budibase#11622] [Help] [answered]
I created a new query that creates a table called temp_label_table that stores the following values: label_id int, label_group_id int. 
Here is my query:

`CREATE TABLE temp_label_table (
    label_id int,
    label_group_id int
)`

When I try to insert values into this query, I get the following error: "Query Error: invalid input syntax for type integer: ""

This is my insert query:
<img width="706" alt="image" src="https://github.com/Budibase/budibase/assets/9454156/8b64f665-bd47-47c3-b8f8-c927479dd064">

Has anyone encountered this before and could help me out here? Thank you!

**Comment by ConorWebb96** [comment, marked as answer]
Hey @vaishnaray,

I noticed an issue with your table create query: `CREATE TABLE temp_label_table ( label_id int, label_group_id int )`. This doesn't include any reference to a primary key? (I've modified this to include this as you won't be able to fetch the table in Budibase unless it has a primary key.)

```
CREATE TABLE temp_label_table (
    id SERIAL PRIMARY KEY,
    label_id INT,
    label_group_id INT
);
```

As for the rest of this, I wasn't able to replicate. It submitted fine and created the record for me as well when I tested. Below is the whole query, from start to finish. I'm currently on the latest version: **2.9.34**.
<img width="471" alt="Screenshot 2023-08-31 at 10 28 15" src="https://github.com/Budibase/budibase/assets/126772285/67e632f8-e6e7-4425-a570-cb9be47499ac">

**Output**
<img width="1062" alt="Screenshot 2023-08-31 at 10 33 52" src="https://github.com/Budibase/budibase/assets/126772285/64e5ef96-d7fd-4121-9a6c-5f816ce69cf4">

Can you answer a few of the below questions;
- What version of Budibase are you running?
- Are you self-host (If yes what self-hosting version are you using) or are you using cloud?
- What browser are you using? (Likely not relevant but asking regardless.)
- Is your data source MySQL?
- Can you screenshot your entire query structure, so I can see if any of the settings differ from the one I provided above.
- Where you able to fetch that table even though there was no primary key provided?

Thanks ahead of time for the information.

**Reply by vaishnaray** [reply]
Once I added the primary key, I was able to get both the ids into the temp table! I guess the primary key was the only thing that was needed. Thank you!
